main.go

```diff
@@ -31,6 +31,10 @@ func main() {
 	app.Usage = ""
 	app.Version = Version
 	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "helm-binary, b",
+			Usage: "path to helm binary",
```

Comment:
Oh, I somehow missed about this, but do you have any specific reason not to add `Value: "helm",` instead of putting this in many places?

```
if c.String("helm-binary") != "" {
    helm.SetHelmBinary(c.String("helm-binary"))
}
```

Just asking. I'd address it myself even when it turned out we'd better use the default value instead.

Reply:
I don't have a clear reason, but in other words I thought that it would be bad for the default value definition to be scattered in two places.

If we include default values when parsing command arguments, it may be more appropriate to pass them as arguments to execer's constructor.

Reply:
Yep, made a lot of sense! I'll definitely consider doing so. Thanks for your confirmation.